src/objprint/objprint.py

```diff
@@ -337,3 +383,27 @@ def _get_pack_str(
         else:
             s = ", ".join(elems)
             return f"{header}{s}{footer}"
+
+    def _validate_formatter(
```

Comment:
Hmm, I don't think we need this validator. The validator relies on the type annotation and I tend to avoid having run-time check on type annotation for this library. The user should be responsible for their code and we just generate an exception when the code is bad.

Reply:
Sure, I may still keep the obj_type check as it is not relying on type annotation, and is a relatively important property to validate.